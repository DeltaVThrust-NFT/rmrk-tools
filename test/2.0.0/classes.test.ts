import { createNftClassMock } from "./mocks";

describe("rmrk2.0.0 NftClass: Create", () => {
  it("should match snapshot", async () => {
    const nftClass = createNftClassMock();
    expect(await nftClass.create()).toMatchSnapshot();
  });
});
