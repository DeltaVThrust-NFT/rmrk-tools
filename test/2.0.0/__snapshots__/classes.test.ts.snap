// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rmrk2.0.0 NftClass: Create should match snapshot 1`] = `"RMRK::CREATE::2.0.0::%7B%22max%22%3A0%2C%22issuer%22%3A%22D6HSL6nGXHLYWSN8jiL9MSNixH2F2o382KkHsZAtfZvBnxM%22%2C%22symbol%22%3A%22KANARIABIRDS%22%2C%22id%22%3A%22classId%22%2C%22metadata%22%3A%22https%3A%2F%2Fsome.url%22%7D"`;
